src/app/Http/Middleware/PollPermsPresenter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Http\Response;
+
+class PollPermsPresenter
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if ($request->user()->id != $request->poll->presenter()) {
```

Comment:
one poll can be manage by many other people rather than presenter and also it's possible to have more than one presenter

Reply:
Fixed in dde2b70